### ISSUE TITLE
ModuleNotFoundError after installing and attempting to use current version

### ISSUE THREAD

**Issue by iaace-NA** [Canisback/roleML#13]
test script recalculate-lanes.py :
```
from roleml import roleml
```

terminal output through windows 10 powershell:
```
PS C:\Users\?\Desktop\sps-shmikal\strategies> pip3 install roleml
Requirement already satisfied: roleml in j:\python38\lib\site-packages (0.1.14)
Requirement already satisfied: pandas in j:\python38\lib\site-packages (from roleml) (1.0.3)
Requirement already satisfied: numpy>=1.16.2 in j:\python38\lib\site-packages (from roleml) (1.18.1)
Requirement already satisfied: scikit-learn>=0.22.1 in j:\python38\lib\site-packages (from roleml) (0.22.2.post1)
Requirement already satisfied: shapely in j:\python38\lib\site-packages (from roleml) (1.7.0)
Requirement already satisfied: pytz>=2017.2 in j:\python38\lib\site-packages (from pandas->roleml) (2019.3)
Requirement already satisfied: python-dateutil>=2.6.1 in j:\python38\lib\site-packages (from pandas->roleml) (2.8.1)
Requirement already satisfied: joblib>=0.11 in j:\python38\lib\site-packages (from scikit-learn>=0.22.1->roleml) (0.14.1)
Requirement already satisfied: scipy>=0.17.0 in j:\python38\lib\site-packages (from scikit-learn>=0.22.1->roleml) (1.4.1)
Requirement already satisfied: six>=1.5 in j:\python38\lib\site-packages (from python-dateutil>=2.6.1->pandas->roleml) (1.14.0)

PS C:\Users\?\Desktop\sps-shmikal\strategies> py recalculate-lanes.py
Traceback (most recent call last):
  File "recalculate-lanes.py", line 1, in <module>
    from roleml import roleml
  File "J:\Python38\lib\site-packages\roleml\roleml.py", line 8, in <module>
    from .utils import exceptions
ModuleNotFoundError: No module named 'roleml.utils'
```

**Comment by Canisback** [comment]
Known issue, due to the way Python on Windows makes the imports. Try to force the 0.1.13 version, it should work.

**Comment by iaace-NA** [comment]
if I just copy the utils folder from this repo's master, will that solve the issue? because it seems to be the only thing missing in the site-packages folder

**Comment by Canisback** [comment]
No, I need to redo the namespace.

**Comment by Canisback** [comment]
Issue fixed on 0.1.15.
